packages/xo-server-sdn-controller/src/index.js

```diff
@@ -1177,6 +1177,15 @@ class SDNController extends EventEmitter {
     }
 
     const tunnel = this._getHostTunnelForNetwork(host, network.$ref)
```

Comment:
IMHO, it would be easier (and less error-prone) if this function threw instead of of returning `undefined` in case it did not work…

Reply:
`undefined` means that there's no PIF or tunnel for the network on this host.
This is not necessary an error.